From b07e59a08b6ba9a8426c66d019e06d1e0d207f19 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Tue, 12 Dec 2017 09:34:39 +0800
Subject: [PATCH 3076/5242] MLK-16715-10 ARM64: dts: fsl-imx8qxp: change
 properties for USB2

commit  e546d634c1701f40c4a5ecf880ae966d5927bb26 from
https://source.codeaurora.org/external/imx/linux-imx.git

The changes include: compatible, performance tuning parameters,
and delete the property which is dedicated to imx7d.

Acked-by: Jun Li <jun.li@nxp.com>
Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index 0e4059f..cdb4bd2 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -1746,17 +1746,19 @@
 	};
 
 	usbotg1: usb@5b0d0000 {
-		compatible = "fsl,imx7d-usb", "fsl,imx27-usb";
+		compatible = "fsl,imx8qm-usb", "fsl,imx27-usb";
 		reg = <0x0 0x5b0d0000 0x0 0x200>;
 		interrupt-parent = <&wu>;
 		interrupts = <GIC_SPI 267 IRQ_TYPE_LEVEL_HIGH>;
 		fsl,usbphy = <&usbphy1>;
 		fsl,usbmisc = <&usbmisc1 0>;
 		clocks = <&clk IMX8QXP_USB2_OH_AHB_CLK>;
-		phy-clkgate-delay-us = <400>;
-		status = "disabled";
+		ahb-burst-config = <0x0>;
+		tx-burst-size-dword = <0x10>;
+		rx-burst-size-dword = <0x10>;
 		#stream-id-cells = <1>;
 		power-domains = <&pd_conn_usbotg0>;
+		status = "disabled";
 	};
 
 	dma_apbh: dma-apbh@5b810000 {
-- 
1.7.9.5

